ITEM_en.tsv

```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
 ITEM_20150323_004072	Formine Armor parts. The part that wraps around one's waist.
 ITEM_20150323_004073	Kindling
 ITEM_20150323_004074	Kindling that was carried by monsters. The wood was dried out more than usual and is useful for maintaining a fire.
-ITEM_20150323_004075	A cold body fluid can be rarely obtained from monster. Let's use it to cold the altar.
-ITEM_20150323_004076	Bag of Teopilis
+ITEM_20150323_004075	A rare, cold body fluid that can be found from monsters. Use it to cool the altar.
+ITEM_20150323_004076	Teopilis's Bag
 ITEM_20150323_004077	Picture Fragment
-ITEM_20150323_004078	Empty frame
-ITEM_20150323_004079	Pemidian Outskirt Warp Scroll
-ITEM_20150323_004081	Strong string
-ITEM_20150323_004082	A strong string obtained from monster. It is useful to tie something.
+ITEM_20150323_004078	Empty Frame
+ITEM_20150323_004079	Femidian Outskirts Warp Scroll
+ITEM_20150323_004081	Strong Cord
+ITEM_20150323_004082	A strong cord obtained from a monster. It is useful for tying things.
 ITEM_20150323_004084	Scorpio Venom
 ITEM_20150323_004086	Sweet Herb
 ITEM_20150323_004088	Nutritious Tonic
-ITEM_20150323_004089	A nutritious tonic that the crude liquid which owner gave is diluted.
-ITEM_20150323_004096	Fresh White Radish
-ITEM_20150323_004097	A full grown white radish just picked in the field. It looks well grown.
-ITEM_20150323_004098	Chopped Wood
-ITEM_20150323_004102	First empty bottle
-ITEM_20150323_004104	A bottle filled with monster's spirit fully
-ITEM_20150323_004105	A bottle filled with monster's spirit fully.
-ITEM_20150323_004110	Evil spirit of Keposid
-ITEM_20150323_004113	A bottle filled with Keposid's evil spirit fully
-ITEM_20150323_004114	Second empty bottle
-ITEM_20150323_004117	Head of small stone statue
-ITEM_20150323_004119	Upper body of small stone statue
-ITEM_20150323_004121	Lower body of small stone statue
-ITEM_20150323_004123	Wing of small stone statue
-ITEM_20150323_004125	Sculpture of blunt tree
-ITEM_20150323_004126	Blunt tree to be used a head of tree hammer
-ITEM_20150323_004127	Blunt tree of stick form
-ITEM_20150323_004128	Blunt tree to be used a handle of tree hammer
-ITEM_20150323_004130	Blunt tree to be used to open the tree liquor barrel. The durability is bad as simply made.
-ITEM_20150323_004135	Bitter drink
-ITEM_20150323_004142	Full Poison pot
-ITEM_20150323_004143	A pot full of a monster's poison.
+ITEM_20150323_004089	A nutritious tonic made by diluting the concentrate that the prince gave you with Scorpio Venom.
+ITEM_20150323_004096	Well-Grown White Radish
+ITEM_20150323_004097	A fully grown white radish freshly picked from the fields. It looks healthy, without blemish.
+ITEM_20150323_004098	Roughly Chopped Wood
+ITEM_20150323_004102	First Empty bottle
+ITEM_20150323_004104	A bottle filled with monster spirits
```

Comment:
A bottle filled with monster spirits -> A bottle filled with a monster's energy. *or* A bottle filled with the energy of a monster.